render_silhouette_regmarks.py

```diff
@@ -0,0 +1,132 @@
+
+		# Create square in top left corner
+		regmark_layer.append(Rectangle.new(left=reg_origin_X, top=reg_origin_Y, width=REG_SQUARE_MM, height=REG_SQUARE_MM, id=REGMARK_TOP_LEFT_ID, style='fill:black;'))
+
+		# Create horizontal and vertical lines in group for top right corner
+		top_right_x = reg_origin_X+reg_width
+		top_right_path = [(top_right_x-REG_LINE_MM,reg_origin_Y), (top_right_x,reg_origin_Y), (top_right_x,reg_origin_Y + REG_LINE_MM)]
+		regmark_layer.append(PathElement.new(path=self.points_to_svgd(top_right_path), id=REGMARK_TOP_RIGHT_ID, style=f"fill:none; stroke:black; stroke-width:{REG_MARK_LINE_WIDTH_MM}"))
+
+		# Create horizontal and vertical lines in group for bottom left corner
+		bottom_left_y = reg_origin_Y+reg_length
+		bottom_left_path = [(reg_origin_X+REG_LINE_MM,bottom_left_y), (reg_origin_X,bottom_left_y), (reg_origin_X,bottom_left_y - REG_LINE_MM)]
+		regmark_layer.append(PathElement.new(path=self.points_to_svgd(bottom_left_path), id=REGMARK_BOTTOM_LEFT_ID, style=f"fill:none; stroke:black; stroke-width:{REG_MARK_LINE_WIDTH_MM}"))
+
+		# Safe Area Marker #
+		# This draws the safe drawing area
+		top_left_safearea_origin_x = reg_origin_X+REG_LINE_MM
+		top_left_safearea_origin_y = reg_origin_Y+REG_LINE_MM
+		top_right_safearea_origin_x = reg_origin_X+reg_width-REG_LINE_MM
+		top_right_safearea_origin_y = reg_origin_Y+REG_LINE_MM
+		bottom_left_safearea_origin_x = reg_origin_X+REG_LINE_MM
+		bottom_left_safearea_origin_y = reg_origin_Y+reg_length-REG_LINE_MM
+		safe_area_points = [
+			(top_left_safearea_origin_x-REG_SAFE_AREA_MM,top_left_safearea_origin_y),
+			(top_left_safearea_origin_x,top_left_safearea_origin_y),
+			(top_left_safearea_origin_x,top_left_safearea_origin_y-REG_SAFE_AREA_MM),
+			(top_right_safearea_origin_x,top_right_safearea_origin_y-REG_SAFE_AREA_MM),
+			(top_right_safearea_origin_x,top_right_safearea_origin_y),
+			(top_right_safearea_origin_x+REG_SAFE_AREA_MM,top_right_safearea_origin_y),
+			(top_right_safearea_origin_x+REG_SAFE_AREA_MM,bottom_left_safearea_origin_y+REG_SAFE_AREA_MM),
+			(bottom_left_safearea_origin_x,bottom_left_safearea_origin_y+REG_SAFE_AREA_MM),
+			(bottom_left_safearea_origin_x,bottom_left_safearea_origin_y),
+			(bottom_left_safearea_origin_x-REG_SAFE_AREA_MM,bottom_left_safearea_origin_y),
+		]
+		regmark_layer.append(PathElement.new(path=self.points_to_svgd(safe_area_points, capped=True), id=REGMARK_SAFE_AREA_ID, style='fill:white;stroke:none'))
+
+		# Add some settings reminders to the print layer as a reminder
+		safe_area_note = f"mark distance from document: Left={reg_origin_X}mm, Top={reg_origin_Y}mm; mark to mark distance: X={reg_width}mm, Y={reg_length}mm; "
+		regmark_layer.append(TextElement(safe_area_note, x=f"{(bottom_left_safearea_origin_x+3)}", y=f"{(bottom_left_safearea_origin_y+(REG_SAFE_AREA_MM+reg_origin_Y/2))}", id = REGMARK_NOTES_ID, style=f"font-size:{REG_MARK_INFO_FONT_SIZE_PT}px"))
```

Comment:
WRT `font-size:{REG_MARK_INFO_FONT_SIZE_PT}px` choose either `pt` or `px` for both: name and unit :smile: